src/ts/components/core/Collapse.tsx

```diff
@@ -5,7 +5,7 @@ import React from "react";
 
 interface Props extends BoxProps, DashBaseProps {
     /** Opened state */
-    in: boolean;
+    opened?: boolean;
```

Comment:
I agree that `open` would be better but went with `opened` because it's consistent with similar props in `Modal` and `Drawer` components  
https://www.dash-mantine-components.com/components/modal#modal

Reply:
OK that makes sense, thanks!